src/kernel/capinv.c

```diff
@@ -127,9 +128,9 @@ kmem_unalloc(unsigned long *pte)
 	 */
 	unsigned long old = *pte;
 
-	assert(old & PGTBL_COSKMEM);
+	assert(chal_chal2cos(old) & PGTBL_COSKMEM);
```

Comment:
This needs a name that is relevant for its functionality.  `chal_pgtbl_chal2cos`.  I think it is fine to think of our pctrie support as pgtbls...

However, you'll want to be consistent across your entire pgtbl api with naming.